src/commands/fng.py

```diff
@@ -0,0 +1,49 @@
+from discord import app_commands, Interaction
+
+
+class FNGGroup(app_commands.Group):
+    def __init__(self):
+        super().__init__(name="fng", description="FNG Group")
+
+    @app_commands.command(name="food", description="Outputs the date of the next Friday Night Games with food")
+    async def food(self, interaction: Interaction):
+        import datetime as dt
+
+        #Manually inputting dates
+        dateStack=[dt.datetime(2024,10,25,17), dt.datetime(2024,8,30,17), dt.datetime(2024,7,26,17), dt.datetime(2024,5,31,17), dt.datetime(2024,4,26,17), dt.datetime(2024,3,29,17)]
```

Comment:
There should be spaces either side of `=`. 

The black formatter should've picked it up during the CI workflow but it was misconfigured (should be fixed now though).